src/inputFunctions.f90

```diff
@@ -269,14 +269,14 @@ end subroutine readAndSetInitialConcentrations
   ! ----------------------------------------------------------------- !
   ! Set the concentrations of all species - default to 0.0_DP.
   ! Called in readAndSetInitialConcentrations()
-  subroutine setConcentrations( concSpeciesNames, inputConcentrations, refSpeciesNames, outputConcentrations )
+  subroutine setConcentrations( concSpeciesNames, inputConcentrations, refSpeciesNames, outputSpecies )
     use types_mod
     use storage_mod, only : maxSpecLength
     implicit none
 
     character(len=maxSpecLength), intent(in) :: concSpeciesNames(:), refSpeciesNames(:)
     real(kind=DP), intent(in) :: inputConcentrations(:)
-    real(kind=DP), intent(out) :: outputConcentrations(:)
+    real(kind=DP), intent(out) :: outputSpecies(:)
```

Comment:
I'm not sure that this is a clearer name? The array contains the concentrations of all chemical species. `outputSpecies` suggests it contains a list of species.

Or is this just a find-and-replace error from the renaming of `outputConcentration.config`?

Reply:
find and replace error :)